desktop-widgets/downloadfromdivecomputer.cpp

```diff
@@ -229,15 +229,17 @@ void DownloadFromDCWidget::updateState(states state)
 	currentState = state;
 }
 
+// FIXME! Get the transport list from libdivecomputer!
```

Comment:
not sure I understand the FIXME. Isn't that what we are doing below?

Reply:
Duh. That FIXME is a left-over from something I already fixed.